src/gui/options_imp.cpp

```diff
@@ -677,6 +686,8 @@ void options_imp::loadOptions()
     checkAdditionDialog->setChecked(AddNewTorrentDialog::isEnabled());
     checkAdditionDialogFront->setChecked(AddNewTorrentDialog::isTopLevel());
     checkStartPaused->setChecked(session->isAddTorrentPaused());
+    deleteTorrentBox->setChecked(TorrentFileGuard::deleteAddedTorrents());
+    deleteCancelledTorrentBox->setChecked(TorrentFileGuard::deleteAddedTorrentsIfCancelled());
```

Comment:
And take care disabling it if `TorrentFileGuard::deleteAddedTorrents()` is false.